apps/frontend/src/lib/exports/index.ts

```diff
@@ -4,29 +4,43 @@ import { toast } from "sonner";
 
 // NOTE: the CSV delimiter is a semicolon. If a user-inputted value has a semicolon it's swapped out with a comma to
 // prevent the exported CSV from being corrupted
-function swapSemicolons(data: string | undefined): string {
+function sanitizeCSVExport(data: string | undefined): string {
   if (!data) {
     return "";
   }
-  return data.replaceAll(";", ",");
+  // Delimiter
+  data = data.replaceAll(";", ",");
+
+  // Escape Characters
+  data = data.replaceAll("\n", " ");
+  data = data.replaceAll("\b", " ");
+  data = data.replaceAll("\f", " ");
+  data = data.replaceAll("\r", " ");
+  data = data.replaceAll("\t", " ");
+  data = data.replaceAll("\v", " ");
```

Comment:
I tried to use a single regex for all these options but it messed up the exports. Will keep looking.

Reply:
Nit: I should change the comment to *Escape Sequences